lib/kubernetes-deploy/kubeclient_builder/kube_config.rb

```diff
@@ -1,21 +1,12 @@
 # frozen_string_literal: true
 
-require 'googleauth'
```

Comment:
We actually still need to import this as it's not included by default in Kubeclient (they didn't want to bloat the gem with a dependency that might not be used by all importers)